Dockerfile.swarm

```diff
@@ -21,9 +21,10 @@ ENV PUBLIC_URL=$public_url
 RUN yarn build:$environment
 
 FROM nginx:1.19.0-alpine AS final
+ARG environment
 WORKDIR /app
 RUN rm /etc/nginx/conf.d/default.conf
-COPY conf/site-swarm.conf /etc/nginx/conf.d/site.conf
+COPY conf/site-$environment.conf /etc/nginx/conf.d/site.conf
```

Comment:
If you made this change, the configuration using for swarm will change from [site-swarm.conf](https://github.com/FromDoppler/doppler-webapp/blob/master/conf/site-swarm.conf) to [site-production.conf](https://github.com/FromDoppler/doppler-webapp/blob/master/conf/site-production.conf) and we do not want it.